### DISCUSSION TITLE
22.0.2

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1481] [General]
## Releases Notes for 22.0.2
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.2/draw.io-22.0.2-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.2/draw.io-22.0.2-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.2/draw.io-universal-22.0.2.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.2/drawio-amd64-22.0.2.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.2/drawio-x86_64-22.0.2.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v22.0.2/drawio-x86_64-22.0.2.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 22.0.2](https://github.com/jgraph/drawio/blob/v22.0.2/ChangeLog). All changes from 22.0.2 are added in this build.
- Fixes #517 
- Fixes #1479 

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v22.0.2'>22.0.2</a>.</em>

**Comment by volnet** [comment]
Thank you for your attention and support. 
I have downloaded version 22.0.2, and the issue #1479 has been resolved. 
Very much appreciated!
